src/hooks/useSignedUrl.ts

```diff
@@ -0,0 +1,32 @@
+import { useQuery } from 'react-query';
```

Comment:
What about `useContentProtection` as a name for this hook?  It doesn't take or return a url as a parameter so the url naming is a bit misleading.

Reply:
Yes, sounds good. I have no preference here